src/cim/components/common/ValidatedConditionAlert.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { Alert, AlertVariant } from '@patternfly/react-core';
+import { StatusCondition } from '../../types';
+
+type ValidationAlertProps = {
+  condition: StatusCondition<'Validated'>;
+};
+
+const ValidatedConditionAlert = ({ condition }: ValidationAlertProps) => {
+  const { status, message } = condition;
+  if (status === 'True') return null;
+  if (status === 'Unknown') {
+    return <Alert variant={AlertVariant.info} title={message} isInline />;
+  }
+  // const [title, ...messages] = message.split(/[:,]/);
+  const [title, ...messages] = message.split(/:|\.,/);
+  return (
+    <Alert variant={AlertVariant.warning} title={title} isInline>
+      {messages.length && (
```

Comment:
what about not rendering the Alert at all in case of no-messages?

Reply:
I believe that the alert should not be rendered only in ase when the condition status is 'True' which is handled in the component.